src/test/java/org/cactoos/func/FallbackFromTest.java

```diff
@@ -58,8 +59,8 @@ void supportsInheritedException() {
             new FallbackFrom<>(
                 new IterableOf<>(RuntimeException.class),
                 exp -> "RuntimeException fallback #1"
-            ).support(IllegalFormatException.class),
-            new IsEqual<>(2)
+            ).support(new IllegalFormatWidthException(1)),
```

Comment:
@baudoliver7 this looks like a typo in the exception name :)

Reply:
It's not a typo. We need here an instance of IllegalFormatException to test inherited support of RuntimeException. Knowing you could not make an instance of IllegalFormatException, I took its subclass IllegalFormatWidthException.

Reply:
@baudoliver7 makes sense, thx for the explanation